tools/IoTEdgeDevOps/DevOpsLib/JsonPathConverter.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DevOpsLib
+{
+    using System;
+    using System.Linq;
+    using System.Reflection;
+    using Newtonsoft.Json;
```

Comment:
Are you going to have documentation on how the `Newtonsoft.Json`?
like where is the 3rd party repo, its significance on our tool.

Reply:
what does it mean? what documentation for?

Reply:
Oh woops, my brain was processing much faster than what I could type.
I was thinking in term of maintenance if we are relying on this Json Serializer, do you potentially see any maintenance hazard? Any remarks? that a future developer needs to be aware about?

Reply:
I am not aware of any maintenance hazard here.  It is feasible to change any usage of JsonPathConverter to another implementation or another 3rd party library.  Let me know if you have any specific concern here.

Reply:
I don't have a specific concern.